index.html

```diff
@@ -44,7 +44,152 @@ <h1 class="primary">
+                        <ul class="tech-stack">
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">HTML</li> 
+                        </ul>
+                        <div class="btn-container">
+                            <button class="see-project-btn" type="button">See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="project-container">
+                    <div class="img-container"></div>
+                    <div class="project-details">
+                        <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                        <ul class="tech-stack">
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">HTML</li> 
+                        </ul>
+                        <div class="btn-container">
+                            <button class="see-project-btn" type="button">See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="project-container">
+                    <div class="img-container"></div>
+                    <div class="project-details">
+                        <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                        <ul class="tech-stack">
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">HTML</li> 
+                        </ul>
+                        <div class="btn-container">
+                            <button class="see-project-btn" type="button">See project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="heading-text about-me">
+              <h2 class="primary color1">About me</h2>  
+              <p class="supporting-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.
+              </p>
+              <div class="btn-container">
+                <button class="resume-btn">Get my resume</button>
+              </div>
+              <div class="qualifications" >
+                  <div class="skills">
+                    <img src="assets/Languages.svg" alt="Skills">
+                    <div>
+                        <h2 class="skill-title color1">Languages</h2>
```

Comment:
- [ ] Please note that it is a project requirement to have Languages, Frameworks & skills in as list items in a `ul` tag . 😄 

![inst](https://github.com/BHObserver/my-portfolio/assets/99656687/2cfb26cb-d0d1-47c7-af16-38d3befaa8ca)